classes/User/User.mjs

```diff
@@ -0,0 +1,110 @@
+import dbDriver from "../../database/driver.mjs"
+import {
+  includeOnly,
+  removeProperties
+} from "../../utilities/removeProperties.mjs"
+
+export class User {
+  constructor(user = null, database = "mongo") {
+    this.database = new dbDriver(database)
+
+    if (user instanceof Object) {
+      this.id = user._id
+    } else {
+      this.id = user
+    }
+    this._fetchUser()
+  }
+
+  async _fetchUser() {
+    let user = await this.database.find({
+      _id: this.id,
+      "@type": "users"
+    })
+    if (Array.isArray(user)) {
```

Comment:
https://github.com/CenterForDigitalHumanities/TPEN-services/blob/main/database/mongo/controller.mjs#L255

it exists

Reply:
I used the dbDriver class in case there was flexibility with db selection, that class has no `getById()` yet. but I'd review this to only use the mongo controller since it's constant